.travis.yml

```diff
@@ -15,6 +15,9 @@ script:
 - make generate
 - if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run
   `make generate`. Please run it and commit the changes"; git status --porcelain; false; fi
+- make bazel-generate
+- if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run
+  `make bazel-generate`. Please run it and commit the changes"; git status --porcelain; false; fi
```

Comment:
can we introduce a `generate-all` command, or addit to `generate`? How time-consuming is this?